app/views/hyrax/collections/_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= simple_form_for @form, html: { class: 'editor' } do |f| %>
+  <div id="descriptions_display">
+    <h2 class="non lower"><%= t('hyrax.collection.form.description') %></h2>
+    <div id="base-terms">
+      <% f.object.primary_terms.each do |term| %>
+        <%= render_edit_field_partial(term, f: f) %>
+      <% end %>
+    </div>
+    <%#= link_to t('hyrax.collection.form.additional_fields'),
+            '#extended-terms',
+            class: 'btn btn-default additional-fields',
```

Comment:
Are we keeping this dead code for a reason? We can always roll back the git history.